### PR TITLE
TINKERPOP-2335 Set node.js support at 10 for 3.5.0

### PR DESCRIPTION
**Pull request by spmallette**
https://issues.apache.org/jira/browse/TINKERPOP-2335

Upgraded frontend plugin and centralized configuration a bit. 

Builds with `mvn clean install`

VOTE +1

### CODE REVIEW

File: gremlin-javascript/pom.xml [hunk 1]
```diff
@@ -29,10 +29,21 @@ limitations under the License.
         <maven.test.skip>false</maven.test.skip>
         <skipTests>${maven.test.skip}</skipTests>
         <gremlin.server.dir>${project.parent.basedir}/gremlin-server</gremlin.server.dir>
+        <npm.version>6.14.6</npm.version>
+        <node.version>v10.22.0</node.version>
```
Comment (jorgebay) [inline gremlin-javascript/pom.xml:33]: Good idea to extract it into a property.

File: gremlin-javascript/src/main/javascript/gremlin-javascript/package.json [hunk 2]
```diff
@@ -40,4 +40,4 @@
   "engines": {
     "node": ">=6"
```
Comment (jorgebay) [inline gremlin-javascript/src/main/javascript/gremlin-javascript/package.json:41]: We should state `"node": ">= 10"` in the package.json

**Review by jorgebay — COMMENTED** [review]
A couple of comments below.

In the meantime, I'll try it locally.

File: gremlin-javascript/src/main/javascript/gremlin-javascript/package.json (hunk 2 above)
Comment (spmallette) [reply gremlin-javascript/src/main/javascript/gremlin-javascript/package.json]: oops - shouldn't have missed that - fixed now

**Review by jorgebay — APPROVED** [review]
I've tested it locally 👍 

Looks good, VOTE +1

**Pull request merged**